subsys/net/ip/ipv6.c

```diff
@@ -36,6 +36,8 @@
 #include "rpl.h"
 #include "net_stats.h"
 
+#define NET_BUF_TIMEOUT MSEC(100)
```

Comment:
Perhaps this should have a less generic name, but e.g. namespaced one, e.g. ND_NET_BUF_TIMEOUT ?

Reply:
Depending on the bearer, use-case, runtime context, it could be wise to let the actual user to set what he needs through Kconfig.

Reply:
Having some config option would be nice but we could end up having multiple config options for various timeouts. I would like to avoid that as that would be very confusing to end user.

For the actual timeout value, I think almost every value is equally good. I mean that if the system is short on buffers (which typically means heavily loaded system), then any timeout value would be equally good. On the other hand, if there is a buffer leak, then any timeout value would not change anything.

So I suggest we go with one hard coded value atm and change this if we really need to.

Reply:
> So I suggest we go with one hard coded value atm and change this if we really need to.

I'm +1 for that. However, I'd still recommend a more distinctive name for such a define.